single_tech_samples/azuresql/scripts/deploy_azure_pipelines_04_multi_stage_predeploy_test.sh

```diff
@@ -6,11 +6,23 @@
 ###############
 # Deploy Pipelines: multi-stage predeploy
 
+echo "Deploying resources for multi-stage with predeployment test pipeline into $RESOURCE_GROUP_NAME"
+
+# Deploy Keyvault
+keyvault_name="mdwsamplekeyvault"
```

Comment:
This needs a random str at the end because KV names need to be globally unique, so this will fail for the next person running this. We can think about adding a consistent random str for the entire deployment -- that is generate in parent deploy.sh script) as it looks like we need this in multiple places in the deploy script (like sql server name).